packages/portal/docs/style/BootstrapVue/Alert.md

```diff
@@ -0,0 +1,21 @@
+Customised BootstrapVue alert component.
+
+Docs including more features and variants: https://bootstrap-vue.org/docs/components/alert
+
+Used by:
+- AlertMessage
+- InfoMessage
+- SearchFilters (Newspaper/WWI api filter alert)
+
+Default variant
+```jsx
+  <b-alert show dismissible class="mb-4">
+    This is a default info variant alert and it is configured to be dismissible
+  </b-alert>
+  <b-alert
+    show
+    variant="dark"
+  >
+    This is a dark variant alert and it is configured to be dismissible
```

Comment:
**Not** dismissible, no?

Reply:
Indeed. Updated to read 'not be dismissable'.